### DISCUSSION TITLE
Method Konnco\FilamentImport\Actions\ImportAction::table does not exist.

### DISCUSSION THREAD

**Discussion by Malebestia** [konnco/filament-import#131] [Q&A]
Hello everyone,
I need to use the ImportAction inside a RelationManager, unfortunately the following error appears:

"Method Konnco\FilamentImport\Actions\ImportAction::table does not exist."

advice?

**Comment by marco76tv** [comment]
same error

**Comment by dennyvik** [comment]
Up... any solution yet?

**Comment by zhinea** [comment]
any update?

**Reply by dennyvik** [reply]
The original ImportAction class is extend Action that design for a header action in normal Filament Resource. Meanwhile when using them in RelationManager it will treated as a table action. 
My solution to solve this is, I copy the ImportAction class of this package to App/Classes/ImportAction.php
Then i change the Action it extend to Filament Table Action, This is work for me.

```php
<?php

namespace App\Classes;

use Filament\Tables\Actions\Action; //replace the original Action that not for Table use

//keep the rest as original

```

Then in the Relation manager where i want to use this Import feature, i am using class from App\Classes

```php
use App\Classes\ImportAction;

class InvoiceListsRelationManager extends RelationManager
{
   public function table(Table $table): Table
    {
           return $table
                   ->headerActions([ 
                            ImportAction::make()
                                    ->label('Import Excel'),
                   ])
    }
}

```

**Reply by Dipterocarpaceae** [reply]
It's works for me, thank you
